cfgov/v1/jinja2/v1/template_debug.html

```diff
@@ -0,0 +1,59 @@
+{% extends "base.html" %}
+
+{% block title %}{{ debug_template_name }}{% endblock %}
+
+{% block css %}
+{{ super() }}
+<style>
```

Comment:
Should we move these to a CSS file some time or does it make more sense to have them in the file here?

Reply:
I stuck these here as this is a proof of concept (and pulled out as its own PR in #5661). I agree it'd be good to have them elsewhere and would welcome any advice as to the best place to put them.